app/models/custom_value/hierarchy_strategy.rb

```diff
@@ -47,14 +65,7 @@ def ar_class
   end
 
   def ar_object(value)
-    item = CustomField::Hierarchy::Item.find_by(id: value.to_s)
-    if item.nil?
-      "#{value} #{I18n.t(:label_not_found)}"
-    elsif item.short.present?
-      "#{item.label} (#{item.short})"
-    else
-      item.label
-    end
+    CustomField::Hierarchy::Item.find_by(id: value.to_s)
```

Comment:
I'm wondering why is `to_s` needed and why `ar_class` is not used:
```suggestion
    ar_class.find_by(id: value)
```